arch/templates/arch/hint_list.html

```diff
@@ -75,3 +75,15 @@ <h2>Hint {{ hint.number }}%</h2>
     </a>
   </div>
 {% endblock layout-content %}
+
+<script defer>
+    document.addEventListener('keydown', function (event) {
+        if (event.key === 'Escape') {
+            console.log('Escape pressed');
+            const link = document.getElementById('backToArch');
```

Comment:
Finding the link element by `document.getElementById` seems like a bit of a hack, I would just replace the location directly rather than clicking a link. You can for example set `window.location.href` or something like this.